wix/Build.OpenJDK_generic.cmd

```diff
@@ -26,14 +26,37 @@ IF NOT DEFINED JVM SET ERR=8
 IF NOT DEFINED PRODUCT_CATEGORY SET ERR=9
 IF NOT %ERR% == 0 ( ECHO Missing args/variable ERR:%ERR% && GOTO FAILED )
 
+REM This needs tidying up, it's got out of control now
 IF NOT "%ARCH%" == "x64" (
 	IF NOT "%ARCH%" == "x86-32" (
-		IF NOT "%ARCH%" == "x86-32 x64" (
-			IF NOT "%ARCH%" == "x64 x86-32" (
-				ECHO ARCH %ARCH% not supported : valid values : x86-32, x64, x86-32 x64, x64 x86-32
-				GOTO FAILED
-			)
-		)
+        IF NOT "%ARCH%" == "arm64" (
+            IF NOT "%ARCH%" == "x86-32 x64" (
+                IF NOT "%ARCH%" == "x86-32 arm64" (
+                    IF NOT "%ARCH%" == "arm64 x86-32" (
+                        IF NOT "%ARCH%" == "x64 x86-32" (
+                            IF NOT "%ARCH%" == "x64 arm64" (
+                                IF NOT "%ARCH%" == "arm64 x64" (
+                                    IF NOT "%ARCH%" == "x86-32 x64 arm64" (
+                                        IF NOT "%ARCH%" == "x86-32 arm64 x64" (
+                                            IF NOT "%ARCH%" == "arm64 x86-32 x64" (
+                                                IF NOT "%ARCH%" == "arm64 x64 x86-32" (
+                                                    IF NOT "%ARCH%" == "x86-32 x64 arm64" (
+                                                        IF NOT "%ARCH%" == "x64 x86-32 arm64" (
+                                                            ECHO ARCH %ARCH% not supported : valid values : x64, x86-32, arm64, x86-32 x64, x64 x86-32, x86-32 x64 arm64, x86-32 arm64 x64, arm64 x86-32 x64, arm64 x64 x86-32, x86-32 x64 arm64, x64 x86-32 arm64
+                                                            GOTO FAILED
+                                                        )
+                                                    )
+                                                )
+                                            )
+                                        )
+                                    )
+                                )
+                            )
+                        )
+                    )
+                )
+            )
+        )
```

Comment:
Haven't we reached the point where it would make sense considering using something else, like PowerShell?

Reply:
A refactor to PS would be great. I'd prefer to get the arm changes in for now though and work on the PS more slowly.

Reply:
I think we can drop and forget all the logic wich assume we build two different things in one run.
I don't see job launched with two arch.
Same for both jvm. One build for hotspot and openj9 at the same time is no more used.
Same for jre and jdk. Not build in same job run since :
>65e6abcc (Morgan Davies      2020-06-05 09:26:03 +0100  475)                 context.println("We have a JRE. Running another installer for it...")